mandatory/3-function-output.js

```diff
@@ -1,16 +1,15 @@
-// Add comments to explain what this function does. You're meant to use Google!
+// The Math.round() method rounds a number to the nearest integer. This function also multiply youranswer by 10.
 function getRandomNumber() {
   return Math.random() * 10;
 }
 
-// Add comments to explain what this function does. You're meant to use Google!
+// The concat() method returns the new string after merging the existing string with other strings without affecting the existing string.
 function combine2Words(word1, word2) {
   return word1.concat(word2);
 }
 
 function concatenate(firstWord, secondWord, thirdWord) {
-  // Write the body of this function to concatenate three words together.
-  // Look at the test case below to understand what this function is expected to return.
+  return firstWord + " " + secondWord + " " + thirdWord
```

Comment:
You could also make it with js concat method  
return firstWord.concat(' ', secondWord, ' ', thirdWord);

Reply:
Thank you very much for suggestion, will use it next time)